src/AlarmClock-Spec/AlarmClockPresenter.class.st

```diff
@@ -0,0 +1,60 @@
+"
+A presenter for an instance of AlarmClock.
+"
+Class {
+	#name : #AlarmClockPresenter,
+	#superclass : #SpPresenter,
+	#instVars : [
+		'model',
+		'elapsedTimeLabel',
+		'finishedLabel'
+	],
+	#category : #'AlarmClock-Spec'
+}
+
+{ #category : #specs }
+AlarmClockPresenter class >> defaultSpec [
+	^ SpBoxLayout newVertical
+		add: #elapsedTimeLabel
+			withConstraints: [ :constraints | constraints height: self labelHeight ];
+		add: #finishedLabel
+			withConstraints: [ :constraints | constraints height: self labelHeight ];
+		yourself
+]
+
+{ #category : #examples }
+AlarmClockPresenter class >> example1 [
+	self new
+		model: (AlarmClock new pass: 5 minutes; yourself);
+		openWithSpec
+]
+
+{ #category : #accessing }
+AlarmClockPresenter >> elapsedTimeString [
+	^ model elapsedTime asString
```

Comment:
This way of converting Time to String is too detailed: we don't need to show milliseconds.

Reply:
123